reaper/util.py

```diff
@@ -134,7 +134,7 @@ def create_archive(content, arcname, metadata=None, outdir=None):
         if metadata is not None:
             zf.comment = json.dumps(metadata, default=metadata_encoder)
         for fn, fp in files:
-            zf.write(fp, os.path.join(arcname, fn))
+            zf.write(fp, os.path.join(arcname, fn) if rootdir else fn)
```

Comment:
Looks like a potential [tarbomb](https://en.wikipedia.org/wiki/Tar_(computing)#Tarbomb). Why not always keep the directory?